src/server/build.js

```diff
@@ -22,6 +22,8 @@ program
   .option('-s, --static-dir <dir-names>', 'Directory where to load static files from', parseList)
   .option('-o, --output-dir [dir-name]', 'Directory where to store built files')
   .option('-c, --config-dir [dir-name]', 'Directory where to load Storybook configurations from')
+  .option('-d, --db-path [db-file]', 'File where to store addon database JSON file')
```

Comment:
May be description need to be changed to "where load from" or something similar.